card-gallery/script.js

```diff
@@ -236,6 +245,9 @@ function loadCards(tsvData, dataGroup) {
     })
   }
 
+  // Expressive search requires more fancy structured data.
+  // TODO: this should be done once per set release / spreadsheet update and stored as a JSON file instead of us re-parsing the TSV every time someone loads the page.
```

Comment:
I don't want to add build steps or anything, but I think it would be useful to eventually move to a model like this:

1. Bart_KF updates the transcription sheet.
2. Someone on the UW team runs a node script to generate new JSON.
3. When a user loads a page, `Card`s get populated directly from the JSON, speeding up load times.

Reply:
Related note: It looks like Bart has updated the sheet since we last pulled it, because his spreadsheet has the featured issue for events, but that column isn't present in the TSV.